.github/workflows/on-pr-to-main.yml

```
@@ -0,0 +1,39 @@
+name: Test Notebooks on PR to Main
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test-notebooks:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)